content/ja/docs/concepts/services-networking/topology-aware-hints.md

```diff
@@ -0,0 +1,105 @@
+---
+reviewers:
+- robscott
+title: Topology Aware Hint
```

Comment:
need to translate `title` and remove the `reviewers` block.

Reply:
Thank you for your review!  I fix some mistakes.